Sources/MapboxMaps/MapView/Configuration/RenderOptions.swift

```diff
@@ -33,4 +33,6 @@ public struct RenderOptions: Equatable {
     /// If, however, the `MapView` is overlaid with a `UIKit` element which must be pinned to a
     /// particular lat-long, then setting this to `true` will result in better synchronization and less jitter.
     public var presentsWithTransaction: Bool = true
+
+    public init() {}
```

Comment:
I still don't understand that purpose of this. There should be an implied free public init that the class gets

Reply:
Turns out the default is `internal`; this was not obvious. From the [Swift "book"](https://docs.swift.org/swift-book/LanguageGuide/AccessControl.html#//apple_ref/doc/uid/TP40014097-CH41-ID3):

> Swift automatically provides a default initializer without any arguments for any structure or base class that provides default values for all of its properties and doesn’t provide at least one initializer itself.
>
> A default initializer has the same access level as the type it initializes, unless that type is defined as public. **For a type that’s defined as public, the default initializer is considered internal**. If you want a public type to be initializable with a no-argument initializer when used in another module, you must explicitly provide a public no-argument initializer yourself as part of the type’s definition.

Reply:
aaaa gotcha, that is interesting. Thanks for the context